.github/workflows/release-pr.yml

```diff
@@ -8,16 +8,23 @@ on:
         type: choice
         options:
           - aligned-sized
-          - light-macros
-          - light-sparse-merkle-tree
+          - light-bounded-vec
+          - light-circuitlib-rs
+          - light-concurrent-merkle-tree
+          - light-hash-set
           - light-hasher
-          - light-merkle-tree-program
-          - light-psp2in2out
-          - light-psp10in2out
-          - light-psp4in4out-app-storage
-          - light-psp2in2out-storage
-          - light-verifier-sdk
-          - macro-circom
+          - light-indexed-merkle-tree
+          - light-macros
+          - light-merkle-tree-event
+          - light-merkle-tree-reference
+          - light-protocol
+          - light-test-utils
+          - light-user-registry
+          - light-utils
+          - light-wasm-hasher
+          - psp-account-compression
+          - psp-compressed-pda
+          - psp-compressed-token
```

Comment:
seems the changes are not applied throughout yet?
psp- -> light-

plus i think we can keep `psp-account-compression` as just `account-compression`